enginetest/queries/event_queries.go

```diff
@@ -0,0 +1,121 @@
+// Copyright 2023 Dolthub, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package queries
+
+import (
+	"github.com/dolthub/go-mysql-server/sql"
+	"github.com/dolthub/go-mysql-server/sql/types"
+)
+
+// EventTests tests any EVENT related behavior. Events have at least one timestamp value (AT/STARTS/ENDS), so to test
+// SHOW EVENTS and SHOW CREATE EVENTS statements, some tests have those timestamps defined in 2037.
+var EventTests = []ScriptTest{
+	{
+		Name: "Simple CREATE EVENTs with ON SCHEDULE EVERY",
```

Comment:
Need a little more variety in creation statements here, make sure we're covering all the possibilities